src/contracts/marketplace.ts

```diff
@@ -196,6 +196,13 @@ export class Marketplace implements UpdateableNetwork {
    *
    * @param listingId - the listing id
    * @returns either a direct or auction listing
+   *
+   * @remarks Create and manage auctions in your marketplace.
+   * @example
+   * ```javascript
+   * const listingId = "1";
+   * const listing = await contract.getListing(listingId);
```

Comment:
```suggestion
   * const listing = await contract.getListing(1);
```

Reply:
the way i did it is consistent with other example snippets we have and IMO more clear what is happening